bluebutton-data-model-codegen/src/main/java/gov/hhs/cms/bluebutton/data/model/codegen/MappingSpec.java

```diff
@@ -291,6 +293,26 @@ public MappingSpec setHeaderEntityTransientFields(String... headerEntityTransien
 		return this;
 	}
 	
+	/**
+	 * @return the fields in {@link #getHeaderEntity()} that should be marked as
+	 *         {@link Transient}
```

Comment:
Is this comment right? Because it looks like you're sticking "unhashedHicn" in here, which I'd guess isn't transient?